tests/virtualization/external/prepare.pm

```diff
@@ -29,7 +29,12 @@ sub run {
             assert_script_run(qq(echo "$ip $guest" >> /etc/hosts));
         }
     } else {
-        assert_script_run "echo \"\$(dig +short $virt_autotest::common::guests{$_}->{ip}) $_ # virtualization\" >> /etc/hosts" foreach (keys %virt_autotest::common::guests);
+        foreach my $guest (keys %virt_autotest::common::guests) {
+            my $vm_name = $virt_autotest::common::guests{$guest}->{vm_name};
+            my $ip = script_output(qq(ssh -o StrictHostKeyChecking=no Administrator\@win2k19.qa.suse.cz 'powershell "get-vm -Name $vm_name | select -ExpandProperty networkadapters | select ipaddresses"' | grep -oE '[0-9]+.[0-9]+.[0-9]+.[0-9]+'));
```

Comment:
There is a hardcode in ssh command, suggest to set the server name as a variable in openqa for instead. So that it can make the change easily once the server updated.

Reply:
On line 27 hyperV guest is hard coded too. 
I suggest we create a new ticket and fix hyperV and VMware domains in a new PR

Reply:
agree